README.md

```diff
@@ -75,6 +75,51 @@ And then execute:
     <%= content_for :render_async %>
     ```
 
+## Advanced usage
+
+`render_async` takes two arguments, `path` and `html_options`.
+
+* `path` is the ajax-capable controller action you're looking to call via `get`. e.g. `comments_stats_path`, `posts_path`, etc.
+* `html_options` is an optional hash that gets passed to a rails `javascript_tag`, to drop html tags into the `script` element.
+
+Example utilizing `html_options` with a `nonce`:
+
+    ```ruby
```

Comment:
Can you just loose the indent here, it doesn't look right in preview :(

![screen shot 2017-07-06 at 9 58 47 am](https://user-images.githubusercontent.com/3028124/27901287-c92803b6-6231-11e7-9327-c001f3bcd02c.png)